.eslintrc.json

```diff
@@ -15,12 +15,25 @@
   "extends": [
     "eslint:recommended",
     "plugin:prettier/recommended",
-    "plugin:@typescript-eslint/eslint-recommended"
+    "plugin:@typescript-eslint/recommended"
   ],
   "rules": {
     "no-unused-vars": "off",
     "@typescript-eslint/no-unused-vars": ["error", {
       "argsIgnorePattern": "^_"
-    }]
-  }
+    }],
+    "@typescript-eslint/ban-types": "off",
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
88 warnings. 

Seems that most of `any` types were added intentionally and fixing them can cause breaking changes.

Reply:
Just added one myself recently for the grid item because we can not know which types the user is going to use at runtime and I needed a usable fallback. This should be totally fine IMO.